npm/install-wrangler.js

```diff
@@ -37,7 +37,7 @@ function downloadAsset(version, platform) {
   }
   mkdirSync(dest);
 
-  const url = `https://cloudflare.works/download-wrangler/${ version }/${ platform }`
+  const url = `https://workers.cloudflare.com/get-npm-wrangler-binary/${ version }/${ platform }`
```

Comment:
nit: can we call it `/get-wrangler` instead? i can see a future where we want to download binaries from something other than `npm` (and maybe people wanna use `yarn`) - seems to make sense to just say `get-wrangler` or `get-wrangler-binary`

Reply:
This is used by the npm installer logic in Wrangler, so I still think it's worth keeping the `npm` string.